libpod/runtime_pod_linux.go

```diff
@@ -13,8 +13,14 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// podIsSharing checks if the pod has any kernel namespaces set as shared. A pause container will not be
+// created if no kernel namespaces are shared.
+func podIsSharing(p *Pod) bool {
```

Comment:
I would rename this to `PodSharesNamespaces()` and make it a public method of Pod - it seems like it could be useful for people using the API to know